extensions/wikia/Blogs/css/oasis.scss

```diff
@@ -261,7 +261,7 @@
 	margin: 4px 2px 0;
 }
 
-wpCategoryInput {
+#wpCategoryInput {
```

Comment:
Since it was not working,
and nobody filed a ticket for that
I would actually remove it
as if we add some styles to it, we might brake something else :)